compiler/cpp/src/lexer/lexer.cpp

```diff
@@ -15,23 +15,20 @@ unsigned int MaxLookahead;
 
 // Input data to be worked with
 std::string FileInput;
-int InputStringPos = -1;
+int InputStringPos = 0;
```

Comment:
I hope that this doesn't break anything we aren't aware of. Regression tests would be nice now, maybe you could dive in test frameworks for C++?

Reply:
Yeah. I also had this thought. Maybe also some integration tests with expected compile outputs with GH Actions.

Reply:
Added this to our ToDo list